openXCiOSFrameworkTests/VehicleManagerTest.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  VehicleManagerTest.swift
+//  openXCiOSFrameworkTests
+//
+//  Created by Ranjan, Kumar sahu (K.) on 22/01/18.
+//  Copyright © 2018 Ford Motor Company. All rights reserved.
+//
+
+import XCTest
+@testable import openXCiOSFramework
+
+class VehicleManagerTest: XCTestCase {
+    
+    
+    var scanValueIs : Bool = false
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testExample() {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    }
+    //Vehicle manager Trace file sink test method
+    func testEnableTraceFileSink(){
+        let value = VehicleManager.sharedInstance.enableTraceFileSink("user/path/.Aiims.txt")
```

Comment:
set all the constants and dummy data in setUp() like validTraceFilePath, invalidTraceFilePath, etc